src/vaadin-grid-selection-mixin.html

```diff
@@ -40,7 +41,7 @@
      * Selects the given item.
      *
      * @method selectItem
-     * @param {Object} item The item object
+     * @param {GridItem} item The item object
```

Comment:
`selectedItemd` should also have `@type {Array<GridItem>}`

Reply:
Ping. `grid.selectedItems` is still of type `Object`.

Reply:
Thanks, will fix. 

There will be a lot of changes because of `@private` anyways, marking as a draft.

Reply:
Done.